db/migrate/20170707205938_default_value_for_reduction_subgroup.rb

```diff
@@ -0,0 +1,6 @@
+class DefaultValueForReductionSubgroup < ActiveRecord::Migration[5.1]
+  def change
+    change_column :reductions, :subgroup, :string, :default => "default"
```

Comment:
Isn't this `"_default"` in https://github.com/zooniverse/caesar/pull/95/files#diff-849381ecfa89c187989a4d14c91b6920R20 ?